### PR TITLE
[incubator/spring-cloud-data-flow] Update to new SCDF version 1.5.2

### PR DESCRIPTION
**Pull request by chrisjs**
- update app version to 1.5.2
- update chart version to 0.2.3
- update skipper to 1.0.7
- update metrics to 2.0.0
- update maintainers

**What this PR does / why we need it**:

Update the version of SCDF server, Skipper and Metrics.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/spring-cloud-data-flow/values.yaml [hunk 1]
```diff
@@ -63,6 +63,8 @@ metrics:
   #  requests:
   #    cpu: 0.5
   #    memory: 640Mi
+  metricsUsername: user
+  metricsPassword: password
```
Comment (unguiculus) [inline incubator/spring-cloud-data-flow/values.yaml]: Leave default empty

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by sabbyanandan** [comment]
Hi, @unguiculus. At your convenience, please let us know if you have any other suggestions for us to improve. Thanks for your time.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**